### DISCUSSION TITLE
Welcome to MHA Discussions!

### DISCUSSION THREAD

**Discussion by stephenegriffin** [microsoft/MHA#247] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by toddalbers** [comment]
Hi.  I would try to get into the code and address this.  But, it would probably take me days to get up to speed on it.  Been a while.  I redacted info from the email header below that doesn't need to be provided to demonstrate the issue.  So disregard things like REDACT or redacteddomoan.com etc.

I would like to submit this email header below which the MHA analyzer missed reporting something from that mxtoolbox reports correctly. 

This MHA analyzer left out of its reporting the Reply-to field.  If you put this same header info below into https://mxtoolbox.com/EmailHeaders.aspx it reports it correctly.  Note that the reply-to in the header info below is unicode UTF-8 text which I think is what may have thrown this MHA analyzer off.  Specifically it is the text =?utf-8?B?5pWE5pWy4oGr5r2Q54mp5pWpcg==?= in the header info below.  If you remove the metadata "=?utf-8?B?" and "=?=" from that string you get the raw unicode UTF-8 text of 5pWE5pWy4oGr5r2Q54mp5pWpcg=  paste that into https://www.base64decode.org/  you get 敄敲⁫潐物敩r which is how it renders in the email:  敄敲⁫潐物敩r <redacted.username@redacteddomain.com>.  On mxtoolbox it renders as:  Reply-To  "=?utf-8?B?5pWE5pWy4oGr5r2Q54mp5pWpcg==?=" <redacted.name@redacteddomain.com>.  But, this MHA analyzer leaves Reply-To out of it's results completely.

Try pasting the header below in both of the following and then look at\for the "Reply-To" in both of the results.  You will the difference regarding Reply-To.  Hope this helps.

https://mha.azurewebsites.net/
https://mxtoolbox.com/EmailHeaders.aspx


Received: from S05-REDACT-16.S05.local (00.30.00.00) by
 S05-REDACT-16.S05.local (00.30.00.00) with Microsoft SMTP Server (TLS) id
 15.0.0000.2 via Mailbox Transport; Wed, 17 Mar 2021 12:24:40 -0400
Received: from S05-CAS024.S05.local (00.30.00.00) by
 S05-REDACT-16.S05.local (00.30.00.00) with Microsoft SMTP Server (TLS) id
 15.0.0000.2; Wed, 17 Mar 2021 12:24:40 -0400
Received: from S05-REDACT-14.S05.local (00.30.00.00) by
 S05-REDACT.S05.local (00.30.00.00) with Microsoft SMTP Server id
 15.0.0000.2 via Frontend Transport; Wed, 17 Mar 2021 12:24:40 -0400
Received: from S05-REDACT.S05.local (mx05-0.redacteddomain.com
 [199.000.000.000]) by S05-REDACT-14.S05.local (Postfix) with ESMTP id
 1559REDACT1 for <redacted.username@redacteddomain.com>;
 Wed, 17 Mar 2021 12:24:38 -0400 (EDT)
X-MC-SPF: pass
Received: from mailda.docusign.net (mailda.docusign.net [64.207.219.135])
 by mx00-1.otherredacteddomain.com (Postfix) with ESMTPS id REDACT0000DE for
 <redacted.username@redacteddomain.com>; Wed, 17 Mar 2021 12:24:37 -0400 (EDT)
Received: from CH1FE82.corp.docusign.net (unknown [10.102.101.12]) by
 mailda.docusign.net (Postfix) with ESMTP id REDACT1F0 for
 <redacted.username@redacteddomain.com>; Wed, 17 Mar 2021 16:24:36 +0000 (UTC)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=docusign.net;
 s=mail; t=1615998277; bh=BYas+PMtEGw9lpw9HVeiTIa1c8ovpnErZXYHqz5I7O0=;
 h=Reply-To:From:To:Date:Subject;
 b=NqP4I5RM+cJATJBDOsYsQ/s+bfsz5DudA1SIjq27SeKzY6qnjfVdbzADqnrvNNHMk
 SiSLZ7phD1zcnPmrU4hvMYwjNHVVw5tmRtdMuluHEZUynoxYkG1jNs/008B4zGqCYi
 0WeHh4vK2In7xB7+YRsE8yqTV5BSnV8KFG/Y6mlA=
Received: from docusign.net ([127.0.0.1]) by CH1FE82.corp.docusign.net with
 Microsoft SMTPSVC(10.0.17763.1697); Wed, 17 Mar 2021 09:24:36 -0700
X-OriginalArrivalTime: 17 Mar 2021 16:24:36.0630 (UTC)
 FILETIME=[059B1B60:01D71B4A]
X-MF-Fingerprint: v=2.3 cv=XYwSMLx5 c=1 sm=1 tr=0 b=1
 a=AEndrKS1bHitvXNosMRZng==:117 a=AEndrKS1bHitvXNosMRZng==:17
 a=dESyimp9J3IA:10 a=BIwwpuNhAAAA:8 a=39N2h3pWAAAA:8
 a=ev0uDf31-Vbjn5swnlMA:9 a=QEXdDO2ut3YA:10 a=bbbki6NZh3wA:10
 a=ZhqW9J1hAAAA:8 a=TeuQqM9sAAAA:8 a=We2YDHj9EeZX8y0Q:21 a=_W_S_7VecoQA:10
 a=iD4IVl0RgGEA:10 a=gwblsgKIW8THzilocG5Q:22
X-MF-Information: Contact support for more information
X-MF-ID: 1559A20C4821.A8D27
X-MF-MailCheck: Not scanned
X-MF-IP-Protocol: IPv4
X-MF-SpamCheck: not spam, SpamAssassin (not cached, score=1.212,
 required 5, autolearn=disabled, CMCL_0 0.00, DCC_CHECK 1.10,
 HTML_MESSAGE 0.00, TRACKER_ID 0.10, T_REMOTE_IMAGE 0.01)
X-MF-SpamScore: s
X-MF-From: dse_na2@docusign.net
X-Spam-Status: No
Return-Path: dse_na2@docusign.net
X-MS-Exchange-Organization-Network-Message-Id:
 REDACT-c337-475a-ee58-REDACT
X-RoutingAgent: Treated
x-ms-exchange-organization-authsource: S05-REDACT.S05.local
x-ms-exchange-organization-authas: Anonymous
Sender: DocuSign System <dse_na2@docusign.net>
Received: Wed, 17 Mar 2021 12:24:40 -0400
X-Priority: 3
X-MSMail-Priority: Normal
Thread-Index: AQHXG0REDACTwIviEuhjs6QALgFzw==
Message-ID: <118e895131REDACTa6e80b18a2ccf2@docusign.net>
From: "REDACTED via DocuSign" <dse_na2@docusign.net>
To: "Redacted User Name" <redacted.username@redacteddomain.com>
Subject: Completed: Please DocuSign: Form W-9 Stickboy (1).pdf,
 Agent Referrral - STICKBOY CREATIVE, LLC - r...
Date: Wed, 17 Mar 2021 12:24:36 -0400
Reply-To: "=?utf-8?B?5pWE5pWy4oGr5r2Q54mp5pWpcg==?="
 <redacted.name@redacteddomain.com>
MIME-Version: 1.0
Content-type: Multipart/alternative; charset=utf-8;
 boundary="00B0FEED_message_boundary"
Content-Description: Multipart message

**Comment by toddalbers** [comment]
Um.  I just read the description of this thread.  Your description of this thread makes it obvious that this was intended to be more of a promotion of positive discussion between participants.  And I started it off with a bug report.  In my defense, this is where the "**_Submit feedback on github_**" link on [https://mha.azurewebsites.net/](https://mha.azurewebsites.net/) brought me to.  Sorry to crash your community page with a bug report.  😨  Don't know where else to report this.

**Comment by stephenegriffin** [comment]
![image](https://user-images.githubusercontent.com/6773183/112314326-474e9680-8c7f-11eb-9821-d588df133702.png)
No repro

**Comment by hader2108** [comment]
Thank you for that display email-header site. It's more comfortable than analyse the raw data itself. Can you add one thing to it; highlight the senders address? (Use lookup or whois for this... maybe?) I can look it up and warn it's provider in case of spam. They deal with them reasonably good. Many spammers has been taken down by this way by me. Some payback feels good! It will help the community in it's whole. I know... 1 down, 2 more appear. But if everybody accepts this; more other users will be spammed by 1 person.  How many other take action at this war, the better it will for everybody...... Remember: one can get a fake message from its bank and click on that link.... If they are stupid to fill your account details..... They will rob you blind...... Criminals. A real deal in the EU.
